azure-pipelines.yml

```diff
@@ -77,25 +77,41 @@ stages:
               /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
 
         strategy:
-          matrix:
-            Build_Debug:
-              _BuildConfig: Debug
-              _PublishType: none
-              _SignType: test
-              _DotNetPublishToBlobFeed : false
-            Build_Release:
-              _BuildConfig: Release
-              # PRs or external builds are not signed.
-              ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+          ${{ if ne(variables['System.TeamProject'], 'public') }}:
```

Comment:
You can simplify this by only conditionalizing part of the matrix if you'd like, but looks good otherwise.

Reply:
Yup I tried something earlier and it was being finnicky but got it now thanks.